jib-core/src/main/java/com/google/cloud/tools/jib/api/RegistryImage.java

```diff
@@ -110,7 +110,7 @@ public RegistryImage addCredential(String username, String password) {
    *
    * @param credentialRetriever the {@link CredentialRetriever} to add
    * @return this
-   * @see CredentialRetrieverFactory for useful pre-defined {@link CredentialRetriever}s
+   * @see CredentialRetrieverFactory
```

Comment:
This one seems worth adding the text to the main Javadoc too, like `See {@link CredentialRetrieverFactory} for useful pre-defined {@link CredentialRetriever}s`.